### PR TITLE
EJ2-Documentation-851394-Ug-Documentation-For-Dialog-Editing

### PR DESCRIPTION
**Pull request by harinikrh**

### CODE REVIEW

File: ej2-javascript/grid/editing/dialog-editing.md [hunk 1]
```diff
@@ -42,9 +46,13 @@ In Dialog edit mode, when you start editing the currently selected row data will
...
+The edit dialog in the Grid component allows you to customize its appearance and behavior based on the type of action being performed, such as editing or adding a record. You can modify properties like header text, showCloseIcon, and height to tailor the edit dialog to your specific requirements. Additionally, you can override default localization strings to provide custom text for buttons or other elements within the dialog.
+
+To customize the edit dialog, you need to handle the [actionComplete](../../api/grid/#actioncomplete) event of the Grid component and perform the necessary modifications based on the **requestType** parameter. The **requestType** parameter identifies the type of action being performed, such as **beginEdit** for editing a record or **add** for adding a new record.
 
-In the following example, We have customized the dialog's height and title of the dialog's header and also we have changed the button content name in the dialog's footer.
+>You can refer the Grid [Default text](../global-local/) list for more localization.
+
```
Comment (VigneshNatarajan27) [inline ej2-javascript/grid/editing/dialog-editing.md:54]: use ShowCloseIcon property of dialog in actioncompelte event. also modify in the sample too.

**Review by VigneshNatarajan27 — CHANGES_REQUESTED** [review]
changes proposed

File: ej2-javascript/grid/editing/dialog-editing.md (hunk 1 above)
Comment (harinikrh) [reply ej2-javascript/grid/editing/dialog-editing.md:54]: modified the sample by using the showCloseIcon property

**Review by VigneshNatarajan27 — APPROVED** [review]
changes are fine

**Pull request merged**